pyartcd/pyartcd/pipelines/build_sync.py

```diff
@@ -33,19 +33,19 @@ async def create(cls, *args, **kwargs):
         self = cls(*args, **kwargs)
         self.group_runtime = await GroupRuntime.create(
             self.runtime.config, self.working_dir,
-            self.group, self.assembly
+            self.group, self.assembly, self.doozer_data_path, self.doozer_data_gitref
         )
         return self
 
-    def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool, data_path: str,
+    def __init__(self, runtime: Runtime, version: str, assembly: str, publish: bool, doozer_data_path: str,
```

Comment:
This looks unnecessary, and pollutes the actual change with lots of changed lines with no reason. Isn't `data_path` clear enough?

Reply:
the issue is GroupRuntime.create function will reload the self args, thus in other functions they all use doozer_data_path and if we pass self.data_path it will raise an error with self.doozer_data_path not found